exchanges/hitbtc/hitbtc_websocket.go

```diff
@@ -31,6 +34,22 @@ const (
 	errAuthFailed          = 1002
 )
 
+var subscriptionNames = map[string]string{
+	subscription.TickerChannel:    "Ticker",
+	subscription.OrderbookChannel: "Orderbook",
+	subscription.CandlesChannel:   "Candles",
+	subscription.AllTradesChannel: "Trades",
+	subscription.MyAccountChannel: "Reports",
+}
+
+var defaultSubscriptions = subscription.List{
+	{Enabled: true, Asset: asset.Spot, Channel: subscription.TickerChannel},
+	{Enabled: true, Asset: asset.Spot, Channel: subscription.OrderbookChannel},
+	{Enabled: true, Asset: asset.Spot, Channel: subscription.CandlesChannel, Interval: kline.ThirtyMin, Levels: 100},
```

Comment:
```
[WARN]  | WEBSOCKET | 05/12/2024 16:07:15 | HitBTC - Unhandled websocket message: {"jsonrpc":"2.0","method":"snapshotCandles",
...
```
Not your ~keys~ bug not your ~coins~ fix

Reply:
Spun off into a miniseries #1740